agave_app/renderer.cpp

```diff
@@ -303,7 +303,6 @@ Renderer::reset(int from)
   glEnable(GL_LINE_SMOOTH);
 
   this->m_time.start();
-  this->m_time = this->m_time.addMSecs(-from);
```

Comment:
Any reason not to remove the unused parameter `from` from the Render::reset function?

Reply:
The only reason is to minimize the changes here.  As I recall there was some warning coming from Qt about addMSecs and this was just cleaning it up.  It's really a minor change but you're right to think about whether the from time will ever be needed at all.

Reply:
Looking again, I believe it was that QTime is deprecated in some way and QElapsedTimer is the more appropriate class to use for this kind of measurement.